enterprise/backend/src/metabase_enterprise/sso/integrations/jwt.clj

```diff
@@ -110,19 +110,6 @@
       (sync-groups! user jwt-data)
       {:session session, :redirect-url redirect-url, :jwt-data jwt-data})))
 
-(defn- check-jwt-enabled []
```

Comment:
This was never hit because we check for jwt-enabled and jwt-configured in the dispatch function of the sso-* multimethods.